corehq/apps/geospatial/tasks.py

```diff
@@ -1,5 +1,22 @@
+from corehq.util.decorators import serial_task
+
 from corehq.apps.celery import task
-from corehq.apps.geospatial.utils import CeleryTaskTracker, update_cases_owner
+from corehq.apps.geospatial.const import INDEX_ES_TASK_HELPER_BASE_KEY
+from corehq.apps.geospatial.utils import (
+    get_celery_task_tracker,
+    CeleryTaskTracker,
+    update_cases_owner,
+    get_geo_case_property,
+)
+from corehq.apps.geospatial.management.commands.index_geolocation_case_properties import (
```

Comment:
Perhaps a good alternative would be to move the `es_case_query` function off to the `es.py` file as I feel it would fit well there. This can then be renamed to something like `es_case_query_for_missing_geopoint_val` or something to that effect.

Alternatively, I can also simply create a file in a new "utils" directory to contain just the above helper function. I do feel the first option makes sense enough and is a bit more straightforward, but happy to go in this direction as well if you think it sounds more suitable @kaapstorm.

Reply:
Addressed in [a161b10](https://github.com/dimagi/commcare-hq/pull/35126/commits/a161b103f75702772c189abc5a7901bee1f7fc99).

Reply:
I agree that it does feel awkward that parts of management command being imported in the app.
I am not sure about the hesitation of moving it to app utils, though, alternate could be tasks/utils or management/utils.